test/helpers/lsp.cc

```diff
@@ -241,7 +239,7 @@ void assertResponseError(int code, string_view msg, const LSPMessage &response)
         "Expected a response message with error `{}: {}`, but received:\n{}", code, msg, response.toJSON());
 }
 
-void assertNotificationMessage(const LSPMethod expectedMethod, const LSPMessage &response) {
+void assertNotificationMessage(LSPMethod expectedMethod, const LSPMessage &response) {
```

Comment:
clang-tidy helpfully informed me that qualifying an enum argument with `const` doesn't actually change anything.